### PR TITLE
Revert infinite scroll and library.more changes (Trac 53765)

### PR DESCRIPTION
**Pull request by hellofromtonya**
Reverts [52287-52288]
Partial revert of [52167] - reverts the `library.more()` logic

Trac ticket: https://core.trac.wordpress.org/ticket/53765

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by SergeyBiryukov — APPROVED** [review]

File: src/js/media/controllers/replace-image.js [hunk 1]
```diff
@@ -90,24 +90,28 @@ ReplaceImage = Library.extend(/** @lends wp.media.controller.ReplaceImage.protot
...
-		this.updateSelection();
+		this.frame.on( 'content:render:browse', this.updateSelection, this );
+
 		Library.prototype.activate.apply( this, arguments );
 	},
 
+	/**
+	 * @since 5.9.0
+	 */
+	deactivate: function() {
```
Comment (anthonyburchell) [inline src/js/media/controllers/replace-image.js:101]: Hi @hellofromtonya ! Is this function still needed in this PR? I'm not seeing where it is being used. Maybe it is part of another PR. :) If you can confirm this function, the rest is good to go in my testing.

Comment (hellofromtonya) [reply src/js/media/controllers/replace-image.js:101]: @anthonyburchell See comment https://core.trac.wordpress.org/ticket/53765#comment:51

Comment (hellofromtonya) [reply src/js/media/controllers/replace-image.js:101]: @anthonyburchell Asked why this is here. Here's the answer https://core.trac.wordpress.org/ticket/53765#comment:69.

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/52384.

**Pull request closed without merging**